nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment-api/src/main/java/org/nuxeo/ecm/platform/comment/api/CommentConstants.java

```diff
@@ -28,6 +28,11 @@
 
     public static final String EVENT_COMMENT_CATEGORY = "commentCategory";
 
+    /**
+     * @since 11.1
+     */
+    public static final String TOP_LEVEL_DOCUMENT = "commented_document";
```

Comment:
```suggestion
    public static final String COMMENTED_DOCUMENT = "commented_document";
```

Reply:
If your suggestion is just about aligning constants names with content I'd rather do it with top level document.
the top level document is the file that is commented. That's how it is named in other methods.
Commented document could be that file, or a comment we are replying to which is more ambiguous